src/classes/Lists.ts

```diff
@@ -19,4 +26,152 @@ export default class Lists extends OneBlinkAPI {
     options = options || {}
     super(options.accessKey, options.secretKey)
   }
+
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const searchParams = {
+   *   limit: 1,
+   *   offset: 0,
+   * }
+   * const { lists, meta } = await lists.searchLists(searchParams)
+   * ```
+   *
+   * @param searchParams Search options
+   */
+  async searchLists(searchParams: ListSearchOptions): Promise<
+    Omit<ListSearchResult, 'formElementDynamicOptionSets'> & {
+      lists: FormTypes.FormElementOptionSet[]
+    }
+  > {
+    if (
+      !searchParams ||
+      typeof searchParams !== 'object' ||
+      typeof searchParams.organisationId !== 'string'
+    ) {
+      return Promise.reject(
+        new TypeError('Must supply "options.organisationId" as a string'),
+      )
+    }
+    const result = await super.searchRequest<ListSearchResult>(
+      basePath,
+      searchParams,
+    )
+    return {
+      meta: result.meta,
+      lists: result.formElementDynamicOptionSets,
```

Comment:
Sorry mate, 1 more...
Think this should be 
```
formElementLists
```
Its what Josh has in the .net sdk and its consistent with lookups `formElementLookups`

Reply:
Yeah sounds good.  I think the return type in the search lookups needs fixing which I will do in the same commit.